src/main/resources/application.yml

```diff
@@ -53,3 +53,10 @@ spring:
   rabbitmq:
     host: ${RABBITMQ_HOST}
     port: ${RABBITMQ_PORT}
+
+keyStore:
+  filePath: "./src/main/resources/consent-manager-keystore.p12"
```

Comment:
Configurable. and comments about how to generate one, and one in test/root which can be used for local development.

Reply:
why should we use test/root keystore for local development??

Reply:
if I don't want to create one by myself :)

Reply:
That's okay if you don't add it now.

You need to figure out how we can add the keystore into dockerimage though